corehq/middleware.py

```diff
@@ -156,7 +157,14 @@ def process_response(self, request, response):
             response['Expires'] = "-1"
             response['Pragma'] = "no-cache"
         else:
+            content_type, _ = mimetypes.guess_type(request.path)
             response['Cache-Control'] = "max-age=31536000"
+            del response['Vary']
```

Comment:
Will these keys definitely always be set? (Do we set them? :blush: I'm being lazy, and not diving in to find out.) If not, `response.pop('maybe-key', None)` might be safer.